src/messages/ui.en.json

```diff
@@ -224,17 +226,23 @@
   "connectMB.USBCompatibility.transferStep.step1": "Open the location to which the firmware was downloaded. Most commonly found in your download folder.",
   "connectMB.USBCompatibility.transferStep.step2": "Drag the file onto the micro:bit on your computer's file explorer.",
   "connectMB.USBCompatibility.transferStep.step3": "Once the file has finished transferring, the micro:bit can be connected using Bluetooth.",
+  "connectMB.USBCompatibility.transferStep.altText": "animation of using drag and drop to program a micro:bit",
```

Comment:
Can this say something useful about what you drag where?

Reply:
Not an issue if that's in the surrounding copy.

Reply:
I think it's covered, and I'd argue if it's not clear from the point 2 in the dialog, then point 2 needs re-wording.

![image](https://github.com/microbit-foundation/ml-trainer/assets/95928279/b249150f-13db-46ef-ae76-ec5170f49152)

Reply:
Added more descriptive text [c6f12ec](https://github.com/microbit-foundation/ml-trainer/pull/125/commits/c6f12ecca81e6915bc4c4fcfd068427740abaa5e).